src/sidebar/store/create-store.js

```diff
@@ -196,7 +186,7 @@ export function createStore(modules, initArgs = [], middleware = []) {
  * @template State
  * @template Actions
  * @template {SelectorMap<State>} Selectors
- * @template RootSelectors
+ * @template [RootSelectors={}]
```

Comment:
The `rootSelectors` property of store modules is optional. In store modules without it, this treats it as if it were an empty object.